api/location/models.py

```diff
@@ -10,5 +11,7 @@ class Location(Base, Utility):
     id = Column(Integer, primary_key=True)
     name = Column(String, nullable=False)
     abbreviation = Column(String, nullable=False)
+    country = Column(Enum('Uganda', 'Kenya', 'Nigeria', name='al_country'))
```

Comment:
How about we put the list on a different section using the python enums, checkout https://stackoverflow.com/questions/36136112/defining-sqlalchemy-enum-column-with-python-enum-raises-valueerror-not-a-valid -use this analogy for specifying timezones and locations on a different class and just pass them to the Enum column, as such it would improve the columns readability